src/tests/stub_fd.cc

```diff
@@ -18,7 +18,7 @@ fde *fde::Table = nullptr;
 int fdNFree(void) STUB_RETVAL(-1)
 void fd_open(int, unsigned int, const char *) STUB
 void fd_close(int) STUB
-void fd_bytes(int, int, unsigned int) STUB
+void fd_bytes(int, int, const FdOps) STUB
```

Comment:
There is no variable here for `const` to matter:

```suggestion
void fd_bytes(int, int, FdOps) STUB
```

Reply:
De-const-ed in to-be-pushed 73b392fa36